images/10-waagent/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM debian:stable-slim
+
+RUN apt-get update && apt-get install -yq --no-install-recommends \
+        ifupdown2 \
+        ca-certificates \
+        openssl \
+        sudo \
+        net-tools \
+        parted \
+        iptables \
+        eject \
+        host \
+        python-setuptools \
+        openssh-server && \
+        apt-get clean && rm -rf /var/lib/apt/lists/*
+
+COPY v2.2.34.tar.gz .
```

Comment:
Use `WALinuxAgent.tar.gz` here so we don't need to maintain the version number in too many places.

Reply:
Looks like you can use `ADD`, it can automatically decompress this file.

Reply:
Already modified